### PR TITLE
Person Task

### PR DESCRIPTION
**Pull request by MeiRR17**
Added their balances

### CODE REVIEW

File: Person.java [hunk 1]
```diff
@@ -1,20 +1,58 @@
...
+    //added object's balance.
+    public void addBalance (double money){
+        balance += money;
+    }
+    //method for printing final balance.
+    public void printBalance (){
+        System.out.println(name + "'s current balance is: " + balance + "$.");
+    }
+    //method for returning value.
+    public double getBalance (){
```
Comment (anz-davar) [inline Person.java:36]: call getBalance also and print result in the main

Comment (anz-davar) [reply Person.java:36]: double b1 = obj1.getBalance();
double b2 = obj2.getBalance();
and print b1 and b2

Comment (MeiRR17) [reply Person.java:36]: should it be ignored by java?

Comment (anz-davar) [reply Person.java:36]: why ignored?

Comment (anz-davar) [reply Person.java:36]: you will print balance

Comment (MeiRR17) [reply Person.java:36]: yeah ok

**Review by anz-davar — APPROVED** [review]

**Pull request merged**